.github/workflows/maven.yml

```diff
@@ -35,18 +35,20 @@ jobs:
        
     steps:
     - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744
-      with:
-        persist-credentials: false
+      with:
+        persist-credentials: false
     - uses: actions/cache@88522ab9f39a2ea568f7027eddc7d8d8bc9d59c8 # v3.3.1
       with:
         path: ~/.m2/repository
         key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
         restore-keys: |
           ${{ runner.os }}-maven-
     - name: Set up JDK ${{ matrix.java }}
-      uses: actions/setup-java@cd89f46ac9d01407894225f350157564c9c7cee2 # v3.12.0
+      uses: actions/setup-java@cd89f46ac9d01407894225f350157564c9c7cee2 # v3.12.0
       with:
         distribution: 'temurin'
         java-version: ${{ matrix.java }}
     - name: Build with Maven
       run: mvn -V --no-transfer-progress -D doclint=none --file pom.xml
+    - name: Maven Verify

```

Comment:
The idea is for GitHub to run the default Maven goal. so edit that instead, so that a dev can run `mvn` on their command line and do the same job a GitHub build does.

Reply:
No problem, will change it today